webserver-configs/nginx.conf

```diff
@@ -32,13 +32,13 @@ server {
 
     ## Begin - Security
     # deny all direct access for these folders
-    location ~* /(.git|cache|bin|logs|backups)/.*$ { return 403; }
+    location ~* /(.git|cache|bin|logs|backups|tests)/.*$ { return 403; }
     # deny running scripts inside core system folders
     location ~* /(system|vendor)/.*\.(txt|xml|md|html|yaml|php|pl|py|cgi|twig|sh|bat)$ { return 403; }
     # deny running scripts inside user folder
     location ~* /user/.*\.(txt|md|yaml|php|pl|py|cgi|twig|sh|bat)$ { return 403; }
     # deny access to specific files in the root folder
-    location ~ /(LICENSE.txt|composer.lock|composer.json|nginx.conf|web.config|htaccess.txt|\.htaccess) { return 403; }
+    location ~ /(LICENSE|composer.lock|composer.json|nginx.conf|web.config|htaccess.txt|\.htaccess) { return 403; }
```

Comment:
LICENSE has been renamed to LICENSE.txt https://github.com/getgrav/grav/blob/develop/LICENSE.txt, can you update this?